orttraining/orttraining/test/python/orttraining_test_transformers.py

```diff
@@ -199,9 +196,116 @@ def get_lr_this_step(global_step):
     def setUp(self):
         self.model_tester = BertModelTest.BertModelTester(self)
 
-    def test_for_pretraining(self):
+    def test_for_pretraining_mixed_precision_all(self):
+        # It would be better to test both with/without mixed precision and allreduce_post_accumulation.
+        # However, stress test of all the 4 cases is not stable at lease on the test machine.
```

Comment:
nit: *least

btw, why is it not stable?

Reply:
if I multiple tests are run in one process, it sometimes fails with a segfault. I still need to investigate it. It is at the backend.